### ISSUE TITLE
Error after the latest update: "not enough space in the context's memory pool" (+ link to a potential fix)

### ISSUE THREAD

**Issue by MStanton3** [oobabooga/text-generation-webui#4681]
### Describe the bug

After the latest update, I can't run Goliath 120b GGUF anymore.
Before the update, I was able to run Goliath 120b with 4096 context without any issues. After updating, I get the following error:

"ggml_new_object: not enough space in the context's memory pool (needed 1638880, available 1638544)"

I get the same error with both llamacpp and llamacpp_HF loaders. I tried to set --n_ctx to 2048, but it didn't help.

Here is the discussion of this error in llamacpp github repository: https://github.com/ggerganov/llama.cpp/issues/4114
Looks like they found a fix, would be great to get it implemented in text-generation-webui.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Run Goliath 120b GGUF using the latest version of text-generation-webui

### Logs

```shell
ggml_new_object: not enough space in the context's memory pool (needed 1638880, available 1638544)
```

### System Info

```shell
NVIDIA RTX 4090 Laptop GPU (16 GB VRAM)
Intel i9-13980HX
64 GB RAM
```

**Comment by oobabooga** [comment]
This is already fixed upstream in llama.cpp (see https://github.com/ggerganov/llama.cpp/commit/bbecf3f415797f812893947998bda4f866fa900e), but it hasn't landed in llama-cpp-python yet. It will be fixed once 0.2.19 gets released.

**Comment by MStanton3** [comment]
Can confirm that everything's working as intended with llama-cpp-python 0.2.19.
Thank you!

I think the issue can be closed now.

**Comment by orenderlawrence** [comment]
This bug has returned.

**Comment by hashms0a** [comment]
> This bug has returned.

ggml_new_object: not enough space in the context's memory pool (needed 49152049888, available 49152049152)
Segmentation fault (core dumped)
